kubemarine/admission.py

```diff
@@ -630,20 +632,18 @@ def manage_pss_enrichment(inventory, cluster):
                 verify_parameter(item, procedure_config["defaults"][item], valid_profiles)
             inventory["rbac"]["pss"]["defaults"][item] = procedure_config["defaults"][item]
     if "exemptions" in procedure_config:
-        for item in procedure_config["exemptions"]:
-            if type(procedure_config["exemptions"][item]) is list:
-                inventory["rbac"]["pss"]["exemptions"][item] = procedure_config["exemptions"][item]
+        default_merger.merge(inventory["rbac"]["pss"]["exemptions"], procedure_config["exemptions"])
     if "namespaces" in procedure_config:
         for namespace in procedure_config["namespaces"]:
             # check if the namespace has its own profiles
-            if type(namespace) is dict:
-                for item in namespace:
-                    # exclude name of namespace
-                    if namespace[item]:
-                        if item.endswith("version"):
-                            verify_version(item, namespace[item], minor_version)
-                        else:
-                            verify_parameter(item, namespace[item], valid_profiles)
+            if isinstance(namespace, dict):
+                profiles = list(namespace.values())[0]
+                for item in profiles:
+                    if item.endswith("version"):
+                        verify_version(item, profiles[item], minor_version)
+                    else:
+                        verify_parameter(item, namespace[item], valid_profiles)
+                raise Exception("Custom labels for each namespace are currently not supported")
```

Comment:
This code currently always fails. I fixed it in the way how it should **probably** work, but raise exception until the behavior is full clarified. MANOPD-82728